rst_files/wald_friedman.rst

```diff
@@ -185,57 +168,24 @@ Activate the project environment, ensuring that ``Project.toml`` and ``Manifest.
 
 .. code-block:: julia
 
-  using Distributions, Plots, LaTeXStrings, LinearAlgebra, QuantEcon, Printf, Interpolations
-
-
-  f0 = pdf.(Ref(Beta(1, 1)), range(0, stop = 1, length = 50))
-  f0 = f0 / sum(f0)
-  f1 = pdf.(Ref(Beta(9, 9)), range(0, stop = 1, length = 50))
-  f1 = f1 / sum(f1)  # Make sure sums to 1
-
-  a = plot([f0 f1],
-      xlabel=L"$k$ Values",
-      ylabel=L"Probability of $z_k$",
-      labels=[L"$f_0$" L"$f_1$"],
-      linewidth=2,
-      ylims=[0.;0.07],
-      title="Original Distributions")
-
-  mix = zeros(50, 3)
-  labels = Vector{String}(undef, 3)
-  p_k = [0.25; 0.5; 0.75]
-  for i in 1:3
-      mix[:, i] = p_k[i] * f0 + (1 - p_k[i]) * f1
-      labels[i] = string(L"$p_k$ = ", p_k[i])
-  end
-
-  b = plot(mix,
-           xlabel = L"$k$ Values",
-           ylabel = L"Probability of $z_k$",
-           labels = labels,
-           linewidth = 2,
-           ylims = [0.;0.06],
-           title = "Mixture of Original Distributions")
-
-  plot(a, b, layout = (2, 1), size = (800, 600))
+    using Distributions, Parameters, Plots, Printf, Random, Roots, StatPlots
 
 .. code-block:: julia
-    :class: test
-
-    @testset "First Plot Tests" begin
-        @test mix[7] ≈ 0.005059526866095018
-        @test all(isequal(0.02), f0)
-        @test f1[10] ≈ 0.0011405494355950728
-    end
 
+    plot(plot(reshape(Beta.(1:2:3, 1:2:3), 2, 1), labels = ["f_0", "f_1"], title = "Original Distributions"),
+         plot(reshape(MixtureModel.(Beta, Ref((p -> (p, p)).(1:2:3)), (p -> [p, one(p) - p]).(0.25:0.25:0.75)), 3, 1),
```

Comment:
This code is clever!... maybe a bit too cleverl

Does the syntax of broadcasting over an anonymous function (e.g. `(p -> [p, one(p) - p]).(0.25:0.25:0.75))`) have the potential to scare some of our target audience?

Reply:
I think we can keep the cleverness, but maybe open up the terseness a little. Something like the following? Also fixes an issue where both subplots had the same metadata. 

```
dists = [Beta(1., 1.), Beta(3., 3.)] # Raw distributions. 
mix(dists, coef1) = MixtureModel(Beta, params.(dists), [coef1, 1-coef1]) # To generate our mixture models. 
mixes = mix.(Ref(dists), [0.25, 0.5, 0.75])

plot(
    plot(dists, labels = ["f_0", "f_1"], title = "Original Distributions"),
    plot(mixes, labels = ["1/4-3/4", "1/2-1/2", "3/4-1/4"], title = "Distribution Mixtures"), 
    ylab = "Density", ylim = (0, 2), layout = (2, 1) # Global settings across both plots
)
```

Reply:
Yes, I never like clever.  Terse is not necessarily a virtue and don't forget point 1 of https://github.com/QuantEcon/lecture-source-jl/blob/master/style.md#basic-principles

I think @arnavs has provided something which we can teach a little easier.

Reply:
I strive to be, hehe. I took from the @arnavs version and terse it a bit.

> Run You Clever Boy, and Remember. - Clara Oswald